xbmc/Application.cpp

```diff
@@ -3156,6 +3158,17 @@ PlayBackRet CApplication::PlayFile(CFileItem item, const std::string& player, bo
   if (item.GetMimeType().empty())
     item.FillInMimeType();
 
```

Comment:
I don't know what a game client is and many oth r devs don't know either. Application should not care about this. This is why we have the IPlayer interface and a player factory. Moving RP internals to the app layer is wrong.